DevelopmentProposals/FixingBugWithNestedComponentListIterator/readme.md

```diff
@@ -0,0 +1,36 @@
+## Project: Address problem/bug with nested loops using `ComponentListIterator`
```

Comment:
I agree for this ' bug fix' (which is not a proposal to decouple the Component from the `ComponentList` data structure) it is not necessary to include this DevProp.